### DISCUSSION TITLE
It stopped while importing the project

### DISCUSSION THREAD

**Discussion by PHANTOM-XOTIC** [Sketchware-Pro/Sketchware-Pro#544] [Q&A]
https://user-images.githubusercontent.com/106108205/185732536-23e195a6-312c-4ab0-a2cc-b4ca19eb5627.mp4

@JavkhlanK 
@hasrat-ali

**Comment by j-px** [comment]
Don't mention (ping) anyone when opening Issues or Discussions, thanks.  
Any way you could share that SWB here?
